### PR TITLE
Changed doc to reflect that Let's Encrypt can now issue wildcard hosts

### PR DESCRIPTION
**Pull request by Hugogerva**

### CODE REVIEW

File: docs/administering/reverse-proxy.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ read more about [wildcard DNS for Sandstorm.](wildcard.md)
 Obtain or generate a key and TLS certificate with `example.com` and `*.example.com` in
 subjectAltName. You can do that by [buying a wildcard HTTPS
 certificate](https://google.com/search?q=cheap+wildcard+ssl), following our [self-signed certificate
-authority instructions](self-signed.md), or using [CloudFlare Origin CA's free wildcard
-certificates.](https://blog.cloudflare.com/cloudflare-ca-encryption-origin/)
+authority instructions](self-signed.md), using [CloudFlare Origin CA's free wildcard
+certificates.](https://blog.cloudflare.com/cloudflare-ca-encryption-origin/) or using [certbot to issue a free Let's Encrypt wildcart certificate](https://certbot.eff.org/faq/#does-let-s-encrypt-issue-wildcard-certificates)
```
Comment (ocdtrekkie) [inline docs/administering/reverse-proxy.md]: s/wildcart/wildcard

File: docs/administering/faq.md [hunk 2]
```diff
@@ -517,60 +517,10 @@ that we provide. Pay special attention to the `RewriteRule` stanzas.
...
-
-One approach is to use `*.example.sandcats.io` for wildcard hostnames, but use a `BASE_URL` of `example.com`
-where `example.com` is certified via Let's Encrypt. However, this would mean that `example.com` contains
-IFRAME elements pointing at `*.example.sandcats.io` and these IFRAMEs may not be able to set cookies within
-the subdomain due to the increasingly-common **third-party cookie-blocking** done by browsers such as Chrome
-and Firefox.
+**Yes !** Let's Encrypt [recently started issuing wildcard certificates] (https://certbot.eff.org/faq/#does-let-s-encrypt-issue-wildcard-certificates), which are needed for Sandstorm
+for all the reasons documented in [our documentation on wildcard DNS.](wildcard.md).
+
+To make it all work, you can follow the instructions in [our documentation on reverse proxy](reverse-proxy.md) to configure nginx and Sandstorm. Just generate a certificate and a private key for both your base url (`example.com`) and the wildcard host you created (`*.example.com`).
```
Comment (ocdtrekkie) [inline docs/administering/faq.md]: @paulproteus would probably be sad you used the word "just" here.

IMHO, if you have a working strategy for using Let's Encrypt with Sandstorm, my preference would be to remove the FAQ entry entirely, and make a separate page with the step-by-step instructions for getting a Let's Encrypt wildcard set up and configuring Sandstorm correctly to work with it.

Comment (Hugogerva) [reply docs/administering/faq.md]: Sure, I'll try to find the time to write it ;) But basically I just reproduced the steps from the reverse proxy page.

**Comment by zenhack** [comment]
Seeing as built-in let's encrypt support just landed, as well as the ability to just upload a cert, the change we want here is probably different now.

**Comment by ocdtrekkie** [comment]
Yeah, I'll keep this open for now until the offending documentation is removed/replaced though lest we forget about the FAQ entries.

**Pull request closed without merging**